src/main/java/htsjdk/variant/vcf/VCFHeaderLineTranslator.java

```diff
@@ -48,13 +50,19 @@
     }
 
     public static Map<String,String> parseLine(VCFHeaderVersion version, String valueLine, List<String> expectedTagOrder) {
-        return mapping.get(version).parseLine(valueLine,expectedTagOrder);
+        return mapping.get(version).parseLine(valueLine,expectedTagOrder,Collections.emptySet());
```

Comment:
IMO better to forward here to avoid code duplication.
```suggestion
        return parseLine(version, valueLine, expectedTagOrder, Collections.emptySet());
```

Reply:
I agree, but my goal was not to break the library interface. So I did not want to simply add a parameter to a function, I was forced to duplicate code. What are your policies when it comes to changing the interfaces?